envcov.sh

```diff
@@ -33,7 +33,7 @@ then
 fi
 
 LCOV_PATH="${XCODECOVERAGE_PATH}/lcov-1.11/bin"
-OBJ_DIR=${OBJECT_FILE_DIR_normal}/${CURRENT_ARCH}
+OBJ_DIR="${OBJECT_FILE_DIR_normal}/${CURRENT_ARCH}"
 
 # Fix for the new LLVM-COV that requires gcov to have a -v paramter
 LCOV() {
```

Comment:
You still need them.  The quotes when the variables are set don't carry over to when they are used.  (I'm trying all this locally with a path that has spaces, trust me.)  Bash variable quoting is weird.  I think the logic is that double quotes are basically used once and then disappear (more or less).